baker/algolia/indexToAlgolia.tsx

```diff
@@ -187,11 +188,10 @@ const getPagesRecords = async () => {
     const pageviews = await Pageview.getViewsByUrlObj()
     const gdocs = await Gdoc.getPublishedGdocs()
     const publishedGdocsBySlug = keyBy(gdocs, "slug")
-    const postsApi = await wpdb
-        .getPosts()
-        .then((posts) =>
-            posts.filter((post) => !publishedGdocsBySlug[`/${post.slug}`])
-        )
+    const postsApi = await wpdb.getPosts(
+        undefined,
+        (post) => !publishedGdocsBySlug[`/${post.slug}`]
```

Comment:
I noticed an existing bug related to this that you might want to address here: https://github.com/owid/owid-grapher/issues/2864